etna/transforms/embeddings/models/tstcc.py

```diff
@@ -293,3 +341,8 @@ def load(cls, path: pathlib.Path) -> "TSTCCEmbeddingModel":
                 obj.embedding_model.load(fn=str(model_path))
 
         return obj
+
+    @staticmethod
+    def list_models() -> List[str]:
+        """Return a list of available pretrained models."""
+        return ["tstcc_medium"]
```

Comment:
Do we really need to name it `tstcc_medium`? Why not just `medium`?

Reply:
I think both models are often used together and this naming is more clear.